### PR TITLE
Provide a new API that returns the vector of available ICU locales us…

### PR DESCRIPTION
**Pull request by clydegerber**
…ing uloc_countAvailable() and uloc_getAvailable().

### CODE REVIEW

File: rust_icu_uloc/src/lib.rs [hunk 1]
```diff
@@ -1244,4 +1262,9 @@ mod tests {
             "azerbaïdjanais (cyrillique, Azerbaïdjan, calendrier=calendrier hébraïque, t=it, usage privé=whatever)"
         );
     }
+    #[test]
+    fn test_get_available_locales() {
```
Comment (filmil) [inline rust_icu_uloc/src/lib.rs:1266]: Is there any fairly stable test for the contents of the returned vectors that you could add?

File: rust_icu_uloc/src/lib.rs [hunk 2]
```diff
@@ -361,6 +362,23 @@ impl ULoc {
         )
     }
 
+    /// Returns the collection of available locales via 'uloc_countAvailable'/'uloc_getAvailable'
```
Comment (filmil) [inline rust_icu_uloc/src/lib.rs:365]: Add a formulaic /// Implements `...` here, for coverage.

Also if you are able can you run `make cov` to update the coverage data?
`make cov` is not hermetic and requires `exuberant-ctags`. If you don't have that package or can't install it, feel free not do it.

File: rust_icu_uloc/src/lib.rs (hunk 1 above)
Comment (clydegerber) [reply rust_icu_uloc/src/lib.rs:1266]: I should be able to test for the existence of a few well-known locales. Will investigate.

File: rust_icu_uloc/src/lib.rs (hunk 2 above)
Comment (clydegerber) [reply rust_icu_uloc/src/lib.rs:365]: Will add the 'Implements' comment and see if I can run the 'make cov'

Comment (clydegerber) [reply rust_icu_uloc/src/lib.rs]: I installed exuberant-ctags but the output in my dev environment (Mac OS) doesn't look correct, so not including that.

**Pull request merged**